syntax = "proto3";

package AndroidConnect;

message Notification {
	int32 NotificationParameters = 1;
	string text = 2;
	string undertext = 3;
	string title = 4;
	string imageName = 5;
	repeated NotificationActions nActions = 6;
	NotificationReply reply = 7;
}

message NotificationActions {
	string content = 1;
	string args = 2;
	string aType = 3;
	int32 isRepliable = 4;
}

message NotificationReply {
	string replyvalue = 2;
}